doc/tutorials/using_vxworks_as_uos.rst

```diff
@@ -0,0 +1,139 @@
+.. _using_vxworks_as_uos:
+
+Using VxWorks as User OS
+########################
+
+This tutorial describes how to run `VxWorks`_ as the User OS on the ACRN hypervisor. We are using Kaby Lake-based
```

Comment:
Let's change this first sentence (so we can delete the next section):
```
`VxWorks`_\* is a real-time proprietary OS designed for use
in embedded systems requiring real-time, deterministic performance.
This tutorial describes how to run VxWorks as the User OS
on the ACRN hypervisor.
```

Reply:
Will change it in next version.